packages/shortest/src/cli/commands/analyze.ts

```diff
@@ -30,27 +28,10 @@ const executeAnalyzeCommand = async (
   options: { force?: boolean } = {},
 ): Promise<void> => {
   const log = getLogger();
-  const cwd = process.cwd();
-  const projectInfo = await getProjectInfo();
-  const supportedFrameworks = projectInfo.data.frameworks.filter((f) =>
-    SUPPORTED_FRAMEWORKS.includes(f.id),
-  );
-
-  if (supportedFrameworks.length === 0) {
-    throw new ShortestError(`No supported framework found`);
-  }
-
-  if (supportedFrameworks.length > 1) {
-    throw new ShortestError(
-      `Multiple supported frameworks found: ${supportedFrameworks.map((f) => f.name).join(", ")}`,
-    );
-  }
-
-  const framework = supportedFrameworks[0].id;
-
-  log.info(`Analyzing ${framework} application structure...`);
+  const supportedFrameworkInfo = await detectSupportedFramework();
+  log.info(`Analyzing ${supportedFrameworkInfo.name} application structure...`);
 
-  const analyzer = new AppAnalyzer(cwd, framework);
+  const analyzer = new AppAnalyzer(supportedFrameworkInfo);
```

Comment:
Introduced some changes to support Next.js apps that are not using the repo's root dir.